.github/workflows/build_wheel.yml

```diff
@@ -3,55 +3,18 @@ name: Build wheel
 on: [push, pull_request]
 
 jobs:
-  wheel_without_test:
-    name: build wheel for ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [windows-latest, macos-latest, ubuntu-latest]
-    env:
-      CIBW_SKIP: "pp27-*win* cp27-*manylinux* pp-*manylinux*"
-      CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
-      CIBW_MANYLINUX_I686_IMAGE: manylinux2014
-    steps:
-    - uses: actions/checkout@v1
-    - uses: actions/setup-python@v1
-      name: Install Python 3.7
-      with:
-        python-version: '3.7'
-
-    - name: Install Visual C++ for Python 2.7
-      if: startsWith(matrix.os, 'windows')
-      run: |
-        choco install vcpython27 -f -y
-
-    - name: "install cibuildwheel"
-      run: pip install cibuildwheel==1.4.1
-
-    - name: build wheel
-      run: cibuildwheel .
-
-    - name: Upload wheels
-      uses: actions/upload-artifact@v1
-      with:
-        name: wheels2
-        path: wheelhouse
-
   wheel:
-    name: build wheel for ${{ matrix.os }}
+    name: build wheel with test for ${{ matrix.os }}
```

Comment:
I'd rename this to just:
```
name: ${{ matrix.os }}
```

...because in the GUI "build wheel test for" is already mentioned

Reply:
ok